Cargo.toml

```diff
@@ -1,4 +1,32 @@
 [package]
 name = "clap"
 version = "2.0.0"
-authors = ["James McGlashan <github@darkfox.id.au>"]
+authors = [
+    "Kevin K. <kbknapp@gmail.com>",
+    "Alexander Kuvaev <alexander@kuvaev.me>",
+    "SungRim Huh <sungrimhuh@gmail.com>",
+    "James McGlashan <github@darkfox.id.au>",
+]
+description = "A simple to use, efficient, and full featured Command Line Argument Parser"
```

Comment:
Ideally this isn't added until we are ready for crates. Adding this _then_ ensures that it isn't accidentally uploaded to crates.io.